src/Symfony/Component/Clock/Tests/ClockAwareTraitTest.php

```diff
@@ -21,11 +22,10 @@ class ClockAwareTraitTest extends TestCase
     public function testTrait()
     {
         $sut = new ClockAwareTestImplem();
-
         $this->assertInstanceOf(DatePoint::class, $sut->now());
 
         $clock = new MockClock();
-        $sut = new $sut();
```

Comment:
I was a bit confused by this. Can someone please explain what was happening here?